src/erl.rs

```diff
@@ -767,6 +767,8 @@ fn var<'a>(name: &'a str, constructor: &'a ValueConstructor, env: &mut Env<'_>)
             _ => atom(record_name.to_snake_case()),
         },
 
+        ValueConstructorVariant::Inline => todo!(),
+
         ValueConstructorVariant::LocalVariable => env.local_var_name(name.to_string()),
```

Comment:
This can be the same as `LocalVariable` I think. `LocalVariable` is the next clause in this match expression so we can do this:
```suggestion
        ValueConstructorVariant::Inline | ValueConstructorVariant::LocalVariable => {
            env.local_var_name(name.to_string())
        }
```